lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -15,67 +13,42 @@ class FeatureCard extends React.Component {
   constructor(props) {
     super(props);
 
-    // These variables shouldn't force a state change & subsequent render
-    this.requests = [];
-    this.campaigns = [];
-    this.campaignsToExclude = setting('dsUser.activity.signups', []);
-    this.campaignsApiPage = 1;
-    this.apiClient = new ApiClient('v1');
-
     this.state = {
       actionDisabled: false,
       campaignIndex: -1,
       isSignedUp: false,
       isLoading: false,
     };
 
+    this.campaigns = [];
+    this.controller = new GalleryController();
```

Comment:
Gotcha. Still feels like it's gotten to the point where this should just always be a controller on the slide then, and regardless of whether its 1 FeatureCard or 3 or more, is just a value somewhere for it to know how many FeatureCards to create. Also, I think it should be renamed to `CampaignsController`... a gallery just happens to be the format it take in on instance for how it shows.